pyscf/lib/mcscf/fci_string.c

```diff
@@ -130,12 +130,7 @@ static int binomial(int n, int m)
                 if (m*2 <= n) {
                         m = n - m;
                 }
```

Comment:
If you are using the recursion relation below, shouldn't these lines also be removed?